hugo/layouts/partials/projects/blog.html

```diff
@@ -0,0 +1,13 @@
+<section class="blog band band_grey" id="blog-container">
```

Comment:
It could be reused the `partials/home/blog.html` once moved to `partials/blog.html`